src/interfaces/swig/shogun.i

```diff
@@ -229,24 +229,13 @@ namespace shogun
 #else // SWIGJAVA
 %template(put) CSGObject::put_vector_or_matrix_from_double_matrix_dispatcher<SGMatrix<float64_t>, float64_t>;
 #endif // SWIGJAVA
-#ifdef SWIGPYTHON
-%template(_get_real) CSGObject::get<float64_t, void>;
-%template(_get_int) CSGObject::get<int32_t, void>;
-%template(_get_real_matrix) CSGObject::get<SGMatrix<float64_t>, void>;
```

Comment:
yes this is cleaner!

Reply:
Should be even cleaner now and easier to add new factories and getters!